screenpy/actions/quietly.py

```diff
@@ -0,0 +1,117 @@
+"""
+Quietly allows for "disabling" logging on successful actions & tests
+"""
+from __future__ import annotations
+
+from typing import Any, TypeVar, Union, overload
+
+from hamcrest.core.base_matcher import Matcher
+from typing_extensions import TypeAlias
+
+from screenpy import settings
+from screenpy.actor import Actor
+from screenpy.pacing import the_narrator
+from screenpy.protocols import Answerable, Performable, Resolvable
+
+T = TypeVar("T")
+
+
+class QuietlyMixin:
+    """
+    Quietly needs to mimic the ducks (i.e. objects) they are wrapping.
+    All of the attributes from the "duck" should be exposed by the Quietly object.
+    """
+
+    def __getattr__(self, key: Any) -> Any:
+        try:
+            return getattr(self.duck, key)
+        except AttributeError as exc:
+            msg = (
+                f"{self.__class__.__name__}({self.duck.__class__.__name__}) "
+                f"has no attribute '{key}'"
+            )
+            raise AttributeError(msg) from exc
+
+
+class QuietlyPerformable(Performable, QuietlyMixin):
+    """Act like the Performable passed in but kink the cable when performing"""
+
+    def perform_as(self, actor: Actor) -> None:
+        with the_narrator.mic_cable_kinked():
+            self.duck.perform_as(actor)
+            if not settings.UNABRIDGED_LOGGING:
+                the_narrator.clear_backup()
```

Comment:
As we discussed in Discord, i think this would be better as an escape hatch outside of the kink:

```python
    def perform_as(self, actor: Actor) -> None:
        if settings.UNABRIDGED_MODE:
            self.duck.perform_as(the_actor)

        with the_narrator.mic_cable_kinked():
            self.duck.perform_as(actor)
```

Reply:
(And then the similar approach for `QuietlyAnswerable` and `QuietlyResolvable`)

Reply:
For the moment I think it make sense to leave this logic in tact.  At least until `Loudly` is implemented.

Reply:
I'm all right with that, yes.